capture-sdk/sdk/src/main/java/net/gini/android/capture/review/multipage/view/ReviewNavigationBarBottomAdapter.kt

```diff
@@ -0,0 +1,75 @@
+package net.gini.android.capture.review.multipage.view
+
+import android.content.DialogInterface.OnClickListener
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import net.gini.android.capture.databinding.GcReviewNavigationBarBottomBinding
+import net.gini.android.capture.view.CustomLoadingIndicatorAdapter
+import net.gini.android.capture.view.DefaultLoadingIndicatorAdapter
+import net.gini.android.capture.view.InjectedViewAdapter
+
+interface ReviewNavigationBarBottomAdapter: InjectedViewAdapter {
```

Comment:
[Suggestion] We can make this interface more abstract. We don't have to tell clients how to show the loading indicator. We can simply declare two methods: `showLoadingIndicator()` and `hideLoadingIndicator()`. In the default implementation we can then use the `OnButtonLoadingIndicatorAdapter` (oh, we have a dependency on PR #117).

Also renaming `onButtonStatus()` to `setContinueButtonEnabled(enabled: Boolean)` makes it clearer what should be done when implementing the interface.

Generally the interface method names should follow the imperative mood to tell clients what we want them to do in each method.

In summary here is my proposal for the interface:
```
interface ReviewNavigationBarBottomAdapter: InjectedViewAdapter {

    fun setOnContinueButtonClickListener(clickListener: View.OnClickListener)

    fun setOnAddPageButtonClickListener(clickListener: View.OnClickListener)

    fun setAddPageButtonVisibility(visibility: Int)

    fun setContinueButtonEnabled(enabled: Boolean)

    fun showLoadingIndicator()

    fun hideLoadingIndicator()
}
```